frontend/src/authentication/ProfilePage.jsx

```diff
@@ -0,0 +1,389 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import {
+  Box,
+  Text,
+  Stack,
+  HStack,
+  Flex,
+  AbsoluteCenter,
+  Button,
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  CheckboxGroup,
+  Checkbox,
+  Radio,
+  RadioGroup,
```

Comment:
personal recommendation, when having a really long list of stuff (like imports in this case), try to have them sorted alphabetically in the long run this makes it easier to find stuff. Vscode has an extension that makes sorting stuff in code easy: https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines

Reply:
Updated the imports in the recent commit: https://github.com/izuchukwumba/peered/pull/8/commits/5fe0755ce9404ede218e0bb4bcfe4fbc765b2e99